src/main/java/de/tum/in/www1/artemis/web/rest/metis/conversation/dtos/ChannelDTO.java

```diff
@@ -134,10 +150,15 @@ public void setTutorialGroupTitle(String tutorialGroupTitle) {
         this.tutorialGroupTitle = tutorialGroupTitle;
     }
 
+    public String getSubType() {
```

Comment:
Why is this a String and not the enum-type?

Reply:
You are right, I can return the enum here.